geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template

```diff
@@ -96,6 +96,31 @@
     <!-- /geomesa.batchwriter -->
 
     <property>
+        <name>geomesa.mapreduce.split.count.absolute</name>
+        <value></value>
+        <description>Set the absolute number of shards instead of basing it on the number of tablet servers.
+            Setting this value overrides geomesa.mapreduce.split.count.tablet.
+
+            Making this value final without setting a value greater than zero may cause unexpected issues.
+        </description>
+        <final>false</final>
+    </property>
+
+    <property>
+        <name>geomesa.mapreduce.split.count.tablet</name>
```

Comment:
probably should be geomesa.mapreduce.split.count.tserver instead of tablet wondering if the better property would be geomesa.mapreduce.splits.tserver.max and have it mean "max number of splits per tserver"